cmd/asset-syncer/utils.go

```diff
@@ -145,6 +148,68 @@ func (r *HelmRepo) Repo() *models.RepoInternal {
 	return r.RepoInternal
 }
 
+func satisfy(chartInput map[string]interface{}, rule *apprepov1alpha1.FilterRuleSpec) (bool, error) {
+	query, err := gojq.Parse(rule.JQ)
+	if err != nil {
+		return false, fmt.Errorf("Unable to parse JQuery: %v", err)
+
```

Comment:
Not sure why `gofmt` didn't remove the extra space - don't care, just looked odd.